applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,1770 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    fig = plt.figure(figsize=(10, 5))
</pre><p>Mostly curious, what happens if we don't specify figsize here? Does it still look okay?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>